SUMMARY.md

```diff
@@ -75,7 +75,7 @@
 * [Overview](deployment/overview.md)
 * [Create an Application](deployment/create-an-application.md)
 * [Generate API Key & Secret](deployment/generate-api-key-and-secret.md)
-* [Deploying the application using CLI](deployment/deploying-the-application-using-cli.md)
+* [Creating and Deploying the application using CLI](deployment/creating&deploying-the-application-using-cli.md)
```

Comment:
I would avoid adding & in urls because it'a character for splitting query params